src/storage/src/hummock/local_version.rs

```diff
@@ -126,8 +126,12 @@ impl LocalVersion {
         &self.pinned_version
     }
 
-    pub fn swap_max_sync_epoch(&mut self, epoch: HummockEpoch) -> Option<HummockEpoch> {
-        if self.max_sync_epoch > epoch {
+    /// Advance the `max_sync_epoch` to at least `new_epoch`.
+    ///
+    /// Return `Some(prev max_sync_epoch)` if `new_epoch > max_sync_epoch`
+    /// Return `None` if `new_epoch <= max_sync_epoch`
+    pub fn advance_max_sync_epoch(&mut self, epoch: HummockEpoch) -> Option<HummockEpoch> {
```

Comment:
nits: name the parameter as `new_epoch` to make it consistent with the documentation.